File: Medii de Proiectare si Programare/HW/MPP_CSharp/MPP_CSharp/Domain/TrackDTO.cs
namespace MPP_CSharp.Domain
{
    public class TrackDTO
    {
        public Track Track { get; set; }
        public string TrackName { get; set; }
        public int NrParticipants { get; set; }

        public TrackDTO(Track track, int nrParticipants)
        {
            Track = track;
            TrackName = track.Name;
            NrParticipants = nrParticipants;
        }
    }
}

File: Medii de Proiectare si Programare/HW/mpp_hw_copy/mpp_hw/TransportModel/src/main/java/transport/model/TrackDTO.java
package transport.model;

public class TrackDTO {
    private Track track;
    private String trackName;

    private int nrParticipants;


    public TrackDTO(Track track, int nrParticipants) {
        this.track = track;
        this.trackName = track.getName();
        this.nrParticipants = nrParticipants;
    }

    public Track getTrack() {
        return track;
    }

    public void setTrack(Track track) {
        this.track = track;
    }

    public int getNrParticipants() {
        return nrParticipants;
    }

    public void setNrParticipants(int nrParticipants) {
        this.nrParticipants = nrParticipants;
    }

    public String getTrackName() {
        return trackName;
    }
}
